backend/src/main/java/com/redhat/cloud/notifications/db/BehaviorGroupResources.java

```diff
@@ -41,14 +41,20 @@ public BehaviorGroup createDefault(BehaviorGroup behaviorGroup) {
 
     @Transactional
     BehaviorGroup create(String accountId, BehaviorGroup behaviorGroup, boolean isDefaultBehaviorGroup) {
+        List<Long> behaviorGroupsSameDisplayNameAlreadyPresent = entityManager.createNamedQuery("BehaviorGroup.countByDisplayNameAndAccountId", Long.class)
+                .setParameter("displayName", behaviorGroup.getDisplayName())
+                .setParameter("accountId", accountId)
+                .getResultList();
+        if (behaviorGroupsSameDisplayNameAlreadyPresent.get(0).intValue() > 0) {
```

Comment:
```suggestion
        Long behaviorGroupsSameDisplayNameAlreadyPresent = entityManager.createNamedQuery("BehaviorGroup.countByDisplayNameAndAccountId", Long.class)
                .setParameter("displayName", behaviorGroup.getDisplayName())
                .setParameter("accountId", accountId)
                .getSingleResult();
        if (behaviorGroupsSameDisplayNameAlreadyPresent > 0) {
```

Reply:
I read about it and wanted to use getSingleResult() as well, but after reading these arguments against it, I decided to use getResultList() even when there is only one result in most of the cases: https://stackoverflow.com/questions/8500031/what-is-better-getsingleresult-or-getresultlist-jpa

Reply:
I don't see any argument in that discussion in favor of `getResultList()` when the query returns a `count`. Did I miss something?

Reply:
good point. the situation changed after adding COUNT(*) to the query. Before this, I would have to add try and catch and wait for an exception to appear when there is either a result or no result. But with count, the situation is different.

Just changed it to getSingleResult()